client/extensions/woocommerce/woocommerce-services/lib/initialize-labels-state/index.js

```diff
@@ -85,7 +114,15 @@ export default data => {
 				isPacked: formData.is_packed,
 				saved: true,
 			},
-			customs: {},
+			customs: {
+				items: customsItemsData,
+				// Ignore validation in all the tariff number fields that are empty so the user doesn't see everything red from the start
+				ignoreTariffNumberValidation: mapValues(
+					customsItemsData,
+					( { tariffNumber } ) => ! tariffNumber
+				),
+				saved: true,
```

Comment:
`saved` is no longer used in the state tree. I'd prefer to see us remove the reliance on `ignoreTariffNumberValidation` and rely on these two user actions instead:
- whether the user has saved the step
- what values the user has entered in the form

And from there decide whether or not to display the validation using selectors. However, it's already a big PR, so ok with `ignoreTariffNumberValidation` getting merged.

If we stick with `ignoreTariffNumberValidation`, `saved` needs to be removed.

Reply:
Removed "saved" in https://github.com/Automattic/wp-calypso/pull/24974/commits/eb8a98263a02f6a712bd246b3393bf4d2b036fc5. You're right, relying in `ignoreTariffNumberValidation` can be confusing, this is a good refactor to do after the feature is released.